js/save.js

```diff
@@ -67,19 +87,42 @@ function importFlowGram(file) {
     }
 }
 
-function openFromFile(file) {
+function importFunction(file) {
+    try {
+        const parsedFile = JSON.retrocycle(JSON.parse(file))
+        if (parsedFile.signingKey !== signingKey) {
+            swal("Flowgramming doesn't support that file!")
+            return
+        }
+        Object.keys(parsedFile.contexts).forEach((functionName) => {
+            addFunctionToTable(functionName)
+        })
+        parentWindowContexts = {
+            ...parentWindowContexts,
+            ...parsedFile.contexts,
+        }
+        window.opener.addNewContext(parsedFile.contexts)
+    } catch (e) {
+        swal('Something went wrong!')
+    } finally {
+        hideLoader()
+    }
+}
+
```

Comment:
`main` should not be shown as imported in function manager. 
![image](https://user-images.githubusercontent.com/41717568/132215230-ec12a5f7-be52-4720-9acd-b44ee396b254.png)